### PR TITLE
docs: ✨ improve readme.md

### PR DESCRIPTION
**Pull request by okineadev**
Now the readme looks awesome 🔥

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,66 @@
...
+4. Choose the link you want to open in your default browser 🎉.
 
 ## Roadmap
 
-- [ ] Support auto parse git repo web links
+- [ ] Support auto-parsing of git repository web links
   - [ ] GitHub
   - [x] CODING
+- [ ] Localization support for multiple languages
+- [ ] Improve link management UI
```
Comment (yunsii) [inline README.md]: I think it should be "Improve links management UI"?

Comment (okineadev) [reply README.md]: Fixed 🫡

**Comment by yunsii** [comment]
@okineadev Are you noticed the lint task error?

**Comment by okineadev** [comment]
> @okineadev Are you noticed the lint task error?

Yes, but this error is really not critical, I would say irrelevant, because I follow the structure of nested elements

**Comment by yunsii** [comment]
I think your IDE does not work well with eslint extension :joy:

**Pull request merged**